src/renderer/Renderer.ts

```diff
@@ -494,17 +494,19 @@ abstract class Renderer {
     const panelSizeObj = flicking.horizontal
       ? { width: panelSize }
       : { height: panelSize };
-    const firstPanelSizeObj = {
-      size: panelSize,
-      height: referencePanel.height,
-      margin: referencePanel.margin
-    };
 
-    if (!flicking.noPanelStyleOverride) {
+    if (flicking.noPanelStyleOverride) {
+      const firstPanelSizeObj = {
+        size: panelSize,
+        height: referencePanel.height,
+        margin: referencePanel.margin
+      };
+      flicking.panels.forEach(panel => panel.resize(firstPanelSizeObj));
+    } else {
       this._strategy.updatePanelSizes(flicking, panelSizeObj);
+      flicking.panels.forEach(panel => panel.resize());
```

Comment:
As `panelsPerView` restricts all panels to have the same size, we can improve the initialization performance by calculating the panel size of only one element.

If you resize all the panels like this, panels will calculate the sizes of all elements.
In that case, `panelsPerView` has no meaning as the user can restrict the panel size not by option but by CSS.

Please change this to calculate the `height` of the element only, also consider the option `horizontal` while doing that.

Reply:
Considering that calculating the height through every element style has a direct impact on performance,
it seems appropriate to use a cached panel as much as possible, but change it to calculate the height only when it is horizontal.